com.unity.ml-agents/Runtime/Actuators/ActionSegment.cs

```diff
@@ -116,19 +116,29 @@ public bool IsEmpty()
             return Array == null || Array.Length == 0;
         }
 
-        /// <inheritdoc/>
+        /// <summary>
+        /// Returns an enumerator that iterates through the ActionSegment.
```

Comment:
These are basically copies of the .net docs, e.g. https://docs.microsoft.com/en-us/dotnet/api/system.collections.ienumerable.getenumerator?view=net-5.0